modules/apigee/main.tf

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  org_id       = try(google_apigee_organization.organization[0].id, "organizations/${var.project_id}")
+  envgroups    = coalesce(var.envgroups, {})
+  environments = coalesce(var.environments, {})
+  instances    = coalesce(var.instances, {})
+}
+
+resource "google_apigee_organization" "organization" {
+  count                                = var.organization == null ? 0 : 1
```

Comment:
Does it make sense to add to the description of var.organization that setting it to null automatically uses the project as organiaation?